css/style.css

```diff
@@ -120,14 +127,72 @@ nav > ul {
   z-index: 0;
 }
 
+.social_media > li {
+  margin: 10px;
+  list-style: none;
+}
+
+.lang > li {
+  height: inherit;
+  display: grid;
+  background-color: #ebebff;
+  color: #3d51ff;
+  border-radius: 8px;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+}
+
+.popupbody .lang > li {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+}
+
 .popup {
-  position: absolute;
+  position: fixed;
   top: 0;
   left: 0;
   width: 100%;
-  height: 700px;
-  background-color: tomato;
-  display: none;
+  height: 100vh;
+  backdrop-filter: blur(7px);
+  background-color: #ebecf094;
+}
```

Comment:
- [ ] Kindly add the overflow to your popup ``overflow-y`` to make it scroll. This will allow seeing all the elements of the popup
![Screenshot from 2022-02-05 00-16-50](https://user-images.githubusercontent.com/65068771/152610742-a785fe1b-4aa3-411d-ad6e-9835e0d99f12.png)